### DISCUSSION TITLE
Does NVIDIA GeForce GTX 1065 Option Nitro 5 laptop work?

### DISCUSSION THREAD

**Discussion by james1507** [kalkmann/Legion-5600H-Hackintosh#12] [Q&A]
I have Laptop Nitro 5 with option AMD
 Ryzen 5 5600H and NVIDIA GeForce GTX 1065 work with this hackintosh ?
![image](https://github.com/kalkmann/Legion-5600H-Hackintosh/assets/106550125/4e741cea-9844-48b9-9f3b-a0e8dca45aee)
![image](https://github.com/kalkmann/Legion-5600H-Hackintosh/assets/106550125/86bb4feb-d821-4930-af11-d7a55b058b41)

**Comment by oldman20** [comment]
I think surely the answer is NO

**Reply by james1507** [reply]
that's sound sad, maybe I'll sell my laptop :D

**Comment by kalkmann** [comment]
Disable nvidia and use https://github.com/ChefKissInc/NootedRed

Em qua., 27 de set. de 2023 11:56, An Nguyen Van ***@***.***>
escreveu:
